Exercises/assignment5.md

```diff
@@ -117,8 +117,8 @@ case 1:
   case 0: { // DOT8
       trace->fu_type = FUType::ALU;
       trace->alu_type = AluType::DOT8;
-      trace->used_iregs.set(rsrc0);
-      trace->used_iregs.set(rsrc1);
+      trace->src_regs.set(rsrc0);
```

Comment:
I don't think this line will compile; could it be this instead?
src_regs[0] = rsrc0;